pvr.vuplus/resources/settings.xml

```diff
@@ -146,6 +146,39 @@
           </constraints>
           <control type="list" format="integer" />
         </setting>
+        <setting id="channelandgroupupdatemode" type="integer" label="30116">
+          <level>0</level>
+          <default>0</default>
+          <constraints>
+            <options>
+              <option label="30117">0</option> <!-- DISABLED -->
+              <option label="30118">1</option> <!-- NOTIFY_AND_LOG -->
+              <!--<option label="30119">2</option>--> <!-- RELOAD_CHANNELS_AND_GROUPS -->
```

Comment:
Was just getting this ready for when I enable automatic reloading of channels. Right now it will just notify. Plan to enable that option in the next PR.

Reply:
k